lib/MusicBrainz/Server/Controller/Root.pm

```diff
@@ -231,6 +233,10 @@ sub begin : Private
         $alert = l('Our Redis server appears to be down; some features may not work as intended or expected.');
         warn "Redis connection to get alert failed: $_";
     };
+    if ($c->user_exists && $c->user->is_banner_editor) {
+        # For banner editors, show a dismissed banner again after 20 hours (MBS-8940)
+        $alert_mtime = max($alert_mtime, time()-20*60*60);
```

Comment:
Makes it appear as if the banner had been updated at most 20 hours previously (for banner message editors only). If the dismissal is older, the banner will reappear.